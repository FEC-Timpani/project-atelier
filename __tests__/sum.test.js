const sum = require('../sum');
import React from 'react';
import renderer from 'react-test-renderer';
import App from '../client/src/index'

describe('Testing Jest to make sure it works', () => {
  test('adds 1 + 3 to equal 4', () => {
    expect(sum(1,3)).toBe(4);
  })

  test('Renders Hello World', () => {
    const app = renderer.create(<App />)
    let tree = app.toJSON();
    expect(tree).toMatchSnapshot();
  })
})

