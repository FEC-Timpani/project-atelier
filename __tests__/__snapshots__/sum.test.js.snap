// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Jest to make sure it works Renders Hello World 1`] = `
<h1
  id="test"
>
  Hello World!
</h1>
`;
